kurtosis-module/impl/module_io/params.go

```diff
@@ -80,32 +86,36 @@ type ExecuteParams struct {
 	Participants []*ParticipantParams `yaml:"participants"`
 
 	// Parameters controlling the settings of the network itself
-	Network *NetworkParams	`yaml:"network"`
+	Network *NetworkParams `yaml:"network"`
 
 	// If set to false, we won't wait for the EL clients to mine at least 1 block before proceeding with adding the CL clients
 	// This is purely for debug purposes; waiting for blockNumber > 0 is required for the CL network to behave as
 	//  expected, but that wait can be several minutes. Skipping the wait can be a good way to shorten the debug loop on a
 	//  CL client that's failing to start.
-	WaitForMining bool			`yaml:"waitForMining"`
+	WaitForMining bool `yaml:"waitForMining"`
 
 	// If set, the module will block until a finalized epoch has occurred.
 	// If `waitForVerifications` is set to true, this extra wait will be skipped.
-	WaitForFinalization bool	`yaml:"waitForFinalization"`
+	WaitForFinalization bool `yaml:"waitForFinalization"`
 
 	// If set to true, the module will block until all verifications have passed
 	WaitForVerifications bool `yaml:"waitForVerifications"`
 
 	// If set, this will be the maximum number of epochs to wait for the TTD to be reached.
-    // Verifications will be marked as failed if the TTD takes longer.
+	// Verifications will be marked as failed if the TTD takes longer.
 	VerificationsTTDEpochLimit uint64 `yaml:"verificationsTTDEpochLimit"`
 
-	// If set, after the merge, this will be the maximum number of epochs wait for the verifications to succeed. 
+	// If set, after the merge, this will be the maximum number of epochs wait for the verifications to succeed.
 	VerificationsEpochLimit uint64 `yaml:"verificationsEpochLimit"`
 
 	// The log level that the started clients should log at
 	ClientLogLevel GlobalClientLogLevel `yaml:"logLevel"`
 }
 
+type BuilderNetworkParams struct {
```

Comment:
Can we add a comment explaining what this is? Folks look into this file pretty frequently